processing_chain/src/test/groovy/org/orbisgis/processingchain/ProcessingChainTest.groovy

```diff
@@ -0,0 +1,30 @@
+package org.orbisgis.processingchain
+
+import org.junit.jupiter.api.Test
+
+import org.orbisgis.datamanager.h2gis.H2GIS
+import org.orbisgis.processmanager.ProcessMapper
+
+class ProcessingChainTest {
+
+    @Test
+    void runBDTopoProcessingChain(){
```

Comment:
Thanks to the method from the previous comment, you can add :
``` groovy
@EnabledIfSystemProperty(named = "test.processingchain", matches = "true")
```
to disable the test if the resources are not present.

Reply:
I add it right before the @test line since I had a flag if I put it right after the void (inside the {} of the test). It seems to work like that...